### PR TITLE
Add typescript to database

### PR DESCRIPTION
**Pull request by parzhitsky**
Add `typescript` to database.json

### CODE REVIEW

File: database.json [hunk 1]
```diff
@@ -992,12 +986,6 @@
     "repo": "deno-nessie",
     "desc": "A database migration tool for deno."
   },
-  "nessie": {
-    "type": "github",
-    "owner": "halvardssm",
-    "repo": "deno-nessie",
-    "desc": "A database migration tool for deno."
-  },
```
Comment (ry) [inline database.json:1000]: don't remove these

Comment (parzhitsky) [reply database.json:1000]: They are duplicate keys though; is this intended?

image
image

Comment (ry) [reply database.json:1000]: not related to this change

Comment (parzhitsky) [reply database.json]: Duplicates removed in #497

**Pull request merged**